### PR TITLE
fix: Conflicting dockerode version specifications

### PR DESCRIPTION
**Pull request by lowjoel**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by lowjoel** [comment]
Hmm. I can't identify where the conflict in `yarn.lock` is. Is it automatically managed by dependabot and so I shouldn't check those changes in?

File: .changeset/shaggy-camels-remain.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-auth-backend': patch
```
Comment (freben) [inline .changeset/shaggy-camels-remain.md:2]: This doesn't seem to have a change below?

Comment (lowjoel) [reply .changeset/shaggy-camels-remain.md:2]: 🤦 I misread, hang on.

Comment (lowjoel) [reply .changeset/shaggy-camels-remain.md]: Fixed it, sorry for the trouble.

**Review by OrkoHunter — APPROVED** [review]

**Comment by lowjoel** [comment]
I'm not sure if the E2E failure shows this change is bad -- I think it's because it's pulling in an existing version of the binary which uses the older type definition? I can't be sure 🤔

**Comment by freben** [comment]
Hah. Yeah I saw. I'll look into it actually. I think it's completely unrelated complications that get surfaced for odd reasons

**Review by freben — APPROVED** [review]
now it works!

**Pull request merged**